sdk/js-query/src/mock/mock.test.ts

```diff
@@ -232,4 +235,18 @@ describe.skip("mocks match testnet", () => {
       );
     }
   });
+  test("SolAccount to devnet", async () => {
+    const accounts = [
+      "2WDq7wSs9zYrpx2kbHDA4RUTRch2CCTP6ZWaH4GNfnQQ", // Example token in devnet
+      "BVxyYhm498L79r4HMQ9sxZ5bi41DmJmeWZ7SCS7Cyvna", // Example NFT in devnet
+    ];
+
+    const query = new QueryRequest(42, [
+      new PerChainQueryRequest(
+        1,
+        new SolanaAccountQueryRequest("finalized", accounts)
+      ),
+    ]);
+    const resp = await mock.mock(query);
```

Comment:
would be cool if we could generate some bytes to match up this response to, but I realized that testnet isn't live yet so maybe has to be a todo.

Reply:
The Solana tests are currently running against devnet, so I can compare against the sample token and NFT accounts. I added that.